# KinaseSubstrateRelationship

## DB Description
We extracted information from the primary data source, which includes multiple databases
(DB), described in the study conducted by Savage et al., published in Clinical Protocols in 2020.
Furthermore, we integrated data from the most relevant and interconnected databases, as
illustrated in Figures 1 and 2, namely **PhosphoSitePlus**, **SIGNOR**, **PhosphoELM**, **SwissProt** and
**HPRD**. This integration allowed for a comprehensive analysis and synthesis of the information
presented.
The following processing steps have been applied:
1. **Species identification**: Homo sapiens (human) data have been selected.
2. **Gene name mapping with UniProt**: gene names associated with Homo sapiens are mapped
to their corresponding entries in the UniProt database.
3. **Filtering redundant information**: redundant or duplicate information is filtered out to
ensure the dataset contains only unique entries.
These processing steps are crucial for refining the dataset and preparing it for further analysis
or downstream applications.

## Reference

Savage SR, Zhang B. Using phosphoproteomics data to understand cellular signaling: a comprehensive guide to bioinformatics resources. Clin Proteomics. 2020 Jul 11;17:27. doi: 10.1186/s12014-020-09290-x. Erratum in: Clin Proteomics. 2024 Mar 7;21(1):20. PMID: 32676006; PMCID: PMC7353784.
